app/src/main/java/com/example/cooky/ui/recipe/RecipeFragment.kt

```diff
@@ -139,6 +145,28 @@ class RecipeFragment : BaseFragment<FragmentRecipeBinding, RecipeViewModel>() {
         }
     }
 
+    fun handleSaveRecentlyRecipe(){
```

Comment:
lam ngon hon duoc khong?

Reply:
đoạn này xử lý nhiều logic quá mà toàn tách biệt nên e không biết viết sao cho gọn hơn, e cho th recipeIds dùng apply không biết có ổn không, gọn hơn nữa chắc phải tách thêm 1 funtion